dnf.spec

```diff
@@ -1,7 +1,7 @@
 %global hawkey_version 0.7.0
 %global librepo_version 1.7.19
 %global libcomps_version 0.1.8
-%global rpm_version 4.13.0-0.rc1.29
+%global rpm_version 4.14.0
```

Comment:
Khm, for this we need to adapt our rgo builds to produce that version. Unfortunately git-describe returns different one. There is auto-override, but it doesn't work. Would be nice if you will implement version override in yaml+rgo.